### ISSUE TITLE
Tests fail when using `AIRFLOW_VAR_INGESTION_LIMIT=<anything-but-1000>`

### ISSUE THREAD

**Issue by AetherUnbound** [WordPress/openverse-catalog#623]
## Description

PR #555 added an `ingestion_limit` Airflow Variable that could be used to set a (rough) limit on how many records could be ingested for a given provider DAG run. Adding an ingestion limit using environment variables with any value that is not 1000 causes tests to fail.

```
―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― test_build_query_param_default ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
[gw0] linux -- Python 3.10.1 /usr/local/bin/python

    def test_build_query_param_default():
        actual_param = clm.get_next_query_params({})
        expected_param = {"cc": "1", "has_image": "1", "limit": 1000, "skip": 0}
>       assert actual_param == expected_param
E       AssertionError: assert {'cc': '1', '...00, 'skip': 0} == {'cc': '1', '...00, 'skip': 0}
E         Omitting 3 identical items, use -vv to show
E         Differing items:
E         {'limit': 300} != {'limit': 1000}
E         Use -v to get the full diff

tests/dags/providers/provider_api_scripts/test_cleveland_museum.py:50: AssertionError

 tests/dags/providers/provider_api_scripts/test_cleveland_museum.py ⨯                                                                                                                                  8% ▊         

――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― test_build_query_param_increments_offset ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
[gw0] linux -- Python 3.10.1 /usr/local/bin/python

    def test_build_query_param_increments_offset():
        previous_query_params = {"cc": "1", "has_image": "1", "limit": 1000, "skip": 0}
    
        actual_param = clm.get_next_query_params(previous_query_params)
        expected_param = {"cc": "1", "has_image": "1", "limit": 1000, "skip": 1000}
>       assert actual_param == expected_param
E       AssertionError: assert {'cc': '1', '..., 'skip': 300} == {'cc': '1', '... 'skip': 1000}
E         Omitting 3 identical items, use -vv to show
E         Differing items:
E         {'skip': 300} != {'skip': 1000}
E         Use -v to get the full diff

tests/dags/providers/provider_api_scripts/test_cleveland_museum.py:58: AssertionError

 tests/dags/providers/provider_api_scripts/test_cleveland_museum.py ⨯✓✓✓✓✓✓✓✓✓✓✓                                                                                                                     100% ██████████
============================================================================================= short test summary info ==============================================================================================
FAILED tests/dags/providers/provider_api_scripts/test_cleveland_museum.py::test_build_query_param_default - AssertionError: assert {'cc': '1', '...00, 'skip': 0} == {'cc': '1', '...00, 'skip': 0}
FAILED tests/dags/providers/provider_api_scripts/test_cleveland_museum.py::test_build_query_param_increments_offset - AssertionError: assert {'cc': '1', '..., 'skip': 300} == {'cc': '1', '... 'skip': 1000}

Results (4.92s):
      11 passed
       2 failed
         - tests/dags/providers/provider_api_scripts/test_cleveland_museum.py:47 test_build_query_param_default
         - tests/dags/providers/provider_api_scripts/test_cleveland_museum.py:53 test_build_query_param_increments_offset
error: Recipe `_mount-tests` failed on line 67 with exit code 1
error: Recipe `test` failed on line 75 with exit code 1
```

## Reproduction
1. Add `AIRFLOW_VAR_INGESTION_LIMIT=300` to your `.env` file
1. Run `just test`
1. Observe errors

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by AetherUnbound** [comment]
I think there are two ways we could tackle this:
1. Update the tests to use a reasonable maximum, e.g. 100 or 200, so that if a dev sets their value to 300 or more the tests will still pass as expected.
2. Forcibly set the ingestion limit to a high number at the very start of testing. This value should _only_ be changed when the tests are run (or when `just test-session` is run), and should not persist to the webserver or after the test container(s) exit.

**Comment by ramadanomar** [comment]
@AetherUnbound is there a way i can access the `AIRFLOW_VAR_INGESTION_LIMIT` so i can dynamically update each limit on tests to adjust for the users env settings? From what i found until know all limits are hard coded, which seems like a headache if you want to update the amounts of items you test.

**Comment by AetherUnbound** [comment]
Since `AIRFLOW_VAR_INGESTION_LIMIT` is just an environment variable that gets populated from the `.env` file, I think the best approach might be to explicitly set it using `docker-compose`'s `-e` flag in the `mount-tests` recipe:

https://github.com/WordPress/openverse-catalog/blob/2a9647abec0c78734fdd9b22f3be47450e79bbc4/justfile#L77-L82

That _should_ I think override whatever value is in the `.env` file, and it can just be set to some absurdly high number like 1 mil. Another way to tackle this might be by using a plugin like [`pytest-env`](https://github.com/MobileDynasty/pytest-env) which would allow us to set environment variables specifically for testing. Then we could just update the [`pytest.ini` file](https://github.com/WordPress/openverse-catalog/blob/2a9647abec0c78734fdd9b22f3be47450e79bbc4/pytest.ini) with a high number.

I think both of those changes might be more robust than either adjusting the existing tests to a lower number or adjusting specific tests to use a different env variable (with something like [`pytest::monkeypatch`](https://docs.pytest.org/en/7.1.x/how-to/monkeypatch.html)) - both of those would result in us having to set the same thing up for new tests that get added later, whereas the above suggestions would prevent us from having to touch individual tests.

Let me know if I should elaborate more on any of that!

**Comment by ramadanomar** [comment]
I really doubt someone would ever try running unit tests with a sample size bigger than 25k, so i think forcing the enviroment variable with docker compose -e is sufficient. Let me know if i missed something

**Comment by AetherUnbound** [comment]
Your PR is perfect! Glad that worked 🙂

**Comment by zackkrida** [comment]
Fixed in #705
